### ISSUE TITLE
[Feature Request] Add previous and next subtitle to the current selection

### ISSUE THREAD

**Issue by postkevone** [animebook/animebook.github.io#61]
Sometimes when I review my old cards I can't recall the context of the sentence I am reviewing, so having an option to add the previous and next subtitle to the card as context would help a lot.

This was actually a feature in [SubtitleMemorize](https://github.com/kaegi/SubtitleMemorize) that I haven't seen in any other mining app.

**Comment by soamsy** [comment]
Rather than only hitting the plus button, have you tried selecting the text of the subtitle lines you want and then hitting `+`? Unless I'm misunderstanding, this should let you to add as much context you want by drag-and-selecting the text of multiple subtitle lines.

**Comment by postkevone** [comment]
@soamsy Yes, that is a good way to resolve this problem, but it is not feasible to do it for every single card I mine.
